src/graphics.rs

```diff
@@ -186,30 +245,44 @@ fn outside_display(p: Point, width: u32, height: u32, rotation: DisplayRotation)
     false
 }
 
-#[rustfmt::skip]
-//returns index position in the u8-slice and the bit-position inside that u8
-fn find_position(x: u32, y: u32, width: u32, height: u32, rotation: DisplayRotation) -> (u32, u8) {
+fn find_rotation(x: u32, y: u32, width: u32, height: u32, rotation: DisplayRotation) -> (u32, u32) {
     let nx;
     let ny;
     match rotation {
         DisplayRotation::Rotate0 => {
             nx = x;
             ny = y;
-        },
+        }
         DisplayRotation::Rotate90 => {
             nx = width - 1 - y;
             ny = x;
-        } ,
+        }
         DisplayRotation::Rotate180 => {
             nx = width - 1 - x;
             ny = height - 1 - y;
-        },
+        }
         DisplayRotation::Rotate270 => {
             nx = y;
             ny = height - 1 - x;
-        },
+        }
     }
+    (nx, ny)
+}
+
+#[rustfmt::skip]
+//returns index position in the u8-slice and the bit-position inside that u8
+fn find_oct_position(x: u32, y: u32, width: u32, height: u32, rotation: DisplayRotation) -> (u32, bool) {
+    let (nx, ny) = find_rotation(x, y, width, height, rotation);
+    (
+        nx / 2 + (width / 2) * ny,
+        (nx & 0x1) == 0,
+    )
+}
```

Comment:
Maybe add a small description for the true, false case here

Reply:
Added some comments.